.github/workflows/build.yml

```
@@ -1,12 +1,13 @@
 name: Build bootstrap binaries for glibc on Linux
 
-# FILL in
+# Test
 
 on:
   pull_request:
     paths:
       - .github/workflows/build.yml
-      - Library/Homebrew/glibc-bootstrap/**
+      - '**.sh'
```

```suggestion
      - 'build-*.sh'
```

although it's tempting to say you just build unconditionally on every PR.

---

The only reason I didn't do that was to avoid rebuilds if we are just updating the README. But if we don't care about that then I can make this unconditional.